filebeat/Makefile

```diff
@@ -7,6 +7,10 @@ TEST_ENVIRONMENT?=true
 GOX_FLAGS=-arch="amd64 386 arm ppc64 ppc64le"
 ES_BEATS?=..
 
+TARGETS?="windows/amd64 windows/386 darwin/amd64"
+TARGETS_OLD?="linux/amd64 linux/386"
+CGO=true
```

Comment:
I worry it would complicate the lives of the community beats. Also, it's still not enabled for Winlogbeat, but we could overwrite it in that one.

Overall, it seemed like a less risky move to do it per Beat, but it's true that we'll have to be careful to do this whenever we add a new Beat. Any strong arguments to make it the default?